docs/src/pages/demos/progress/DelayingAppearance.js

```diff
@@ -73,7 +78,12 @@ class DelayingAppearance extends React.Component {
         {query === 'success' ? (
           <Typography className={classes.success}>Success!</Typography>
         ) : (
-          <Fade in={query === 'progress'} enterDelay={800}>
+          <Fade
+            in={query === 'progress'}
+            style={{
+              transitionDelay: query === 'progress' ? '800ms' : '0ms',
```

Comment:
The md file also needs to be updated reflect the change in approach.

Reply:
Also the Tabs / FAB integration example needs to be updated: http://localhost:3000/demos/buttons